src/mds/MDSRank.h

```diff
@@ -153,6 +154,7 @@ class MDSRank {
     MDLog        *mdlog;
     MDBalancer   *balancer;
     ScrubStack   *scrubstack;
+    DamageTable  damage_table;
```

Comment:
Any particular reason why this one is a class member instead of following the pointer pattern?